### DISCUSSION TITLE
Extension for Visual Studio Code Marketplace?

### DISCUSSION THREAD

**Discussion by agamirzov** [apple/pkl#659] [Q&A] [answered]
Hi. Thanks for the 🥒 :)

There is a `.vsix` extension that can be downloaded from [here](https://github.com/apple/pkl-vscode/releases/tag/0.17.0).

Though, I am wondering if it was ever planned to add this extension to the VSCode's [Marketplace](https://marketplace.visualstudio.com/)? It would for example enable a better integration with other VSCode features like building containers with `devcontainer.json` (to automatically pull the required extensions). Download link to the `.vsix` file does not work there unfortunately.

Would it be acceptable if the community would add this extension to the Marketplace?

**Comment by bioball** [comment, marked as answer]
This is something that we are working on!

**Reply by cameronbrill** [reply]
hi! let me know if this question should go elsewhere, but is this still being worked on?

**Reply by HT154** [reply]
We are still working toward this!
